lib/page/login/widget/login_text_field_widget.dart

```diff
@@ -22,6 +22,7 @@ class _LoginTextFieldWidgetState extends State<LoginTextFieldWidget> {
   Widget build(BuildContext context) {
     return Container(
       height: 50,
+      margin: EdgeInsets.only(left: 30, right: 30, top: 34),
```

Comment:
You should not add margin here. Instead, you should add margin to be before line 102 of login_page